packages/cli/utils/deploy_cli_template/index.js

```diff
@@ -57,10 +55,10 @@ module.exports = ({ domain, dir, configFn }) => async args => {
     ]
   });
 
-  const workingDir = path.resolve(dir, "tmp");
+  const workingDir = fs.mkdtempSync(path.join(os.tmpdir(), "blossm-"));
```

Comment:
Creates a `blossm-XXXXXX` folder in the `$TMPDIR` folder . On my Mac, it created this:

`/var/folders/xw/jrbqm4xx7b76bd9dkz1jzw4c0000gp/T/blossm-JSPtLR`

Reply:
Awesome